react/index.jsx

```diff
@@ -146,7 +146,7 @@ const MiniCart = ({
   labelButtonFinishShopping,
   iconSize,
   iconLabel,
-  showTotalItemsQty,
+  itemCountMode,
```

Comment:
Careful with this component, this is a breaking change. You're renaming a prop which already exists.

This `.jsx` file in the index for the `minicart` block. If this prop name is changed, it would stop working for users of this block.